src/main/webapp/i18n/en/exerciseChatbot.json

```diff
@@ -0,0 +1,12 @@
+{
+    "artemisApp": {
+        "exerciseChatbot": {
+            "welcome": "Welcome",
+            "popUpMessage": "After you accept the regulations, your chat with IrisBot will start and your data will be shared with the third-party servers. Please decline if you do not wish to chat.",
```

Comment:
Can we be more specific? We should say something like

```
By choosing to continue, you agree that your interactions with IrisBot will be processed by Microsoft and OpenAI, with data transfer occurring outside of our university data center. If you do not agree with these terms, please select 'Decline'. To acknowledge this and begin your chat with IrisBot, press 'Accept'.
```

Reply:
Done